### PR TITLE
Add more colors

### PR DESCRIPTION
**Pull request by czy5074**
https://github.com/javajudt/FavoriteColor/issues/4

Happy hacktoberfest !!

### CODE REVIEW

File: Function.cs [hunk 1]
```diff
@@ -17,7 +17,10 @@ public class Function
 
         private readonly string[] coolColors =
         {
-            "Air Force Blue", 
+            "Air Force Blue",
+            "Antique bronze",
+            "Android green",
```
Comment (javajudt) [inline Function.cs]: Actually, Android Green and Antique Brass have already been added by others. We'll take your Antique Bronze, though!
```suggestion

```

File: Function.cs [hunk 2]
```diff
@@ -17,7 +17,10 @@ public class Function
...
+            "Antique brass",
```
Comment (javajudt) [inline Function.cs]:
```suggestion

```

**Review by javajudt — CHANGES_REQUESTED** [review]
Two of the three you added are duplicates. Just need those removed.

File: Function.cs (hunk 2 above)
Comment (czy5074) [reply Function.cs]: Fixed. Thanks.

**Review by javajudt — APPROVED** [review]
Great!

**Pull request merged**